Makefile

```diff
@@ -73,7 +73,7 @@ vet: $(go_sources) $(GO)
 	touch $@
 
 docker-build:
-	$(OCI_BIN) build -t ${IMAGE_REGISTRY}/${IMAGE_NAME}:${IMAGE_TAG} -f ./cmd/Dockerfile .
+	$(OCI_BIN) build -t ${IMAGE_REGISTRY}/${IMAGE_NAME}:${IMAGE_TAG} -f cmd/Dockerfile .
```

Comment:
Btw in the future if someone uses the `--remote --url=unix://${XDG_RUNTIME_DIR}/podman/podman.sock`
he will need to do this change, for now i am not including it, unless you prefer that I do already
(with the change it should work for all configurations, tested few of them but not all yet)